modules/calib3d/test/test_undistort.cpp

```diff
@@ -1807,4 +1807,78 @@ TEST(Calib3d_initUndistortRectifyMap, regression_14467)
     EXPECT_LE(cvtest::norm(dst, mesh_uv, NORM_INF), 1e-3);
 }
 
+TEST(Calib3d_initInverseRectificationMap, regression_14468)
```

Comment:
> 14468

What does it mean?
Add PR/issue number. Use #20165 as reference on original code.

Reply:
Haha, I incorrectly assumed that was an arbitrary number. Correction has been applied.